Source/astcenc_block_sizes2.cpp

```diff
@@ -849,12 +841,12 @@ static void construct_block_size_descriptor_3d(
 	bsd->decimation_mode_count = decimation_mode_count;
 
 	// then construct the list of block formats
+	int idx = 0;
```

Comment:
We get some name shadowing errors if compiled with `-Werror=shadow` (some of the code lower down also has an `idx` variable). Can we rename the new variable to `packed_idx`, given you use this elsewhere already for this.

Reply:
This and other comments addressed in 6b0dc2b